### PR TITLE
Create Chinese simp. locale

### PR DESCRIPTION
**Pull request by KrasnayaPloshchad**
This will translate DeadLock into Chinese simp.

### CODE REVIEW

**Comment by KrasnayaPloshchad** [comment]
@zerng07 Can you translate DeadLock into Traditional Chinese?

**Comment by zerng07** [comment]
@KrasnayaPloshchad I use GNU/Linux as my working environment, and I only translate programs which are on the platform. Thus, I'm not going to translate it. 

Maybe you can check  L10N_tw group (https://t.me/l10n_tw) on Telegram to find anyone who would like to help.

**Comment by KrasnayaPloshchad** [comment]
@fricyo @Vdragon Can you?

File: DeadLock/Resources/Languages/zhs.xml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  <BarItemHomePage>主页</BarItemHomePage>
+  <BarItemAbout>关于</BarItemAbout>
+  <ClhPath>位置</ClhPath>
+  <ClhStatus>状态</ClhStatus>
+  <ClhOwnership>所有权</ClhOwnership>
+  <MsgUnknown>未知</MsgUnknown>
+  <MsgLocked>已锁定</MsgLocked>
+  <MsgUnlocked>已解锁</MsgUnlocked>
+  <MsgOperationCancelled>操作已取消</MsgOperationCancelled>
+  <MsgLoading>加载中...</MsgLoading>
```
Comment (Artoria2e5) [inline DeadLock/Resources/Languages/zhs.xml]: …

File: DeadLock/Resources/Languages/zhs.xml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  <MsgCouldNotUnlock>无法解锁项目</MsgCouldNotUnlock>
+  <MsgSuccessfullyCopied>复制成功</MsgSuccessfullyCopied>
+  <MsgCouldNotCopy>无法复制项目</MsgCouldNotCopy>
+  <MsgSuccessfullyMoved>移动成功</MsgSuccessfullyMoved>
+  <MsgCouldNotMove>无法移动项目</MsgCouldNotMove>
+  <MsgSuccessfullyRemoved>移除成功</MsgSuccessfullyRemoved>
+  <MsgCouldNotRemove>无法移除项目</MsgCouldNotRemove>
+  <ClhFileName>文件名</ClhFileName>
+  <ClhProcessId>进程 ID</ClhProcessId>
+  <LblVersion>版本:</LblVersion>
```
Comment (Artoria2e5) [inline DeadLock/Resources/Languages/zhs.xml]: ：

File: DeadLock/Resources/Languages/zhs.xml [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  <MsgAdministrator>DeadLock 可能无法不以管理员身份正常运行！</MsgAdministrator>
+  <MsgVersion>版本</MsgVersion>
+  <MsgAvailable>现已推送！</MsgAvailable>
+  <MsgDownloadNewVersion>您是否愿意下载最新版本？</MsgDownloadNewVersion>
+  <MsgLatestVersionAlreadyInstalled>DeadLock 最新版本已安装！</MsgLatestVersionAlreadyInstalled>
+  <TxtAboutCreated>DeadLock 由 DeadLine 创建。</TxtAboutCreated>
+  <TxtAboutImages>图像: small-n-flat by paomedia</TxtAboutImages>
+  <TxtAboutTheme>主题: SyncFusion</TxtAboutTheme>
+  <TxtAboutCopyright>Copyright &#169; CodeDead 2016</TxtAboutCopyright>
+  <TxtAboutTranslation>翻译:</TxtAboutTranslation>
```
Comment (Artoria2e5) [inline DeadLock/Resources/Languages/zhs.xml]: ：

File: DeadLock/Resources/Languages/zhs.xml [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+  <ChbShowAdminWarning>显示管理员警告:</ChbShowAdminWarning>
+  <LblAppearance>外观</LblAppearance>
+  <LblThemeStyle>主题样式:</LblThemeStyle>
+  <LblBorderThickness>边框厚度:</LblBorderThickness>
+  <LblRememberFormSize>记住窗口大小:</LblRememberFormSize>
+  <LblShowDetails>显示详情:</LblShowDetails>
+  <LblLanguage>语言:</LblLanguage>
+  <LblAdvanced>高级</LblAdvanced>
+  <LblAutoRunDeadLock>自动运行 DeadLock:</LblAutoRunDeadLock>
+  <LblWindowsExplorerIntegration>集成到 Windows 文件资源管理器:</LblWindowsExplorerIntegration>
```
Comment (Artoria2e5) [inline DeadLock/Resources/Languages/zhs.xml]: 这年头似乎大家喜欢叫“资源管理器集成”？

File: DeadLock/Resources/Languages/zhs.xml [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+  <BtnReset>重置</BtnReset>
+  <BtnSave>保存</BtnSave>
+  <TbtnOn>开启</TbtnOn>
+  <TbtnOff>关闭</TbtnOff>
+  <TxtUpdater>更新</TxtUpdater>
+  <LblPath>位置:</LblPath>
+  <LblProgress>进程:</LblProgress>
+  <BtnCancel>取消</BtnCancel>
+  <BtnUpdate>更新</BtnUpdate>
+  <MsgCouldNotRestart>无法开始重启会话。不能查出资源锁定方。</MsgCouldNotRestart>
```
Comment (Artoria2e5) [inline DeadLock/Resources/Languages/zhs.xml]: 不能查出？→ 查不出；“锁定方”有点怪

File: DeadLock/Resources/Languages/zhs.xml [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+  <BarItemClearItems>清理项目</BarItemClearItems>
+  <BarItemAutoSizeColumns>自动缩放列宽</BarItemAutoSizeColumns>
+  <BarItemCancelTask>取消任务</BarItemCancelTask>
+  <BarItemDetails>详情</BarItemDetails>
+  <BarItemSettings>设置</BarItemSettings>
+  <BarItemHelp>帮助</BarItemHelp>
+  <BarItemCheckForUpdates>检查更新</BarItemCheckForUpdates>
+  <BarItemHomePage>主页</BarItemHomePage>
+  <BarItemAbout>关于</BarItemAbout>
+  <ClhPath>位置</ClhPath>
```
Comment (Artoria2e5) [inline DeadLock/Resources/Languages/zhs.xml]: 路径？

**Comment by Artoria2e5** [comment]
咦难道是我订阅了吗…

**Comment by KrasnayaPloshchad** [comment]
@Artoria2e5 发错了

**Comment by Artoria2e5** [comment]
啊做繁体啊……最近有点忙很抱歉（.

**Comment by fricyo** [comment]
@KrasnayaPloshchad 是要翻成繁体中文吗?

**Comment by KrasnayaPloshchad** [comment]
@fricyo 是

**Comment by fricyo** [comment]
OK 我翻好了
只是不确定繁体中文的文件名是什么才对

**Comment by KrasnayaPloshchad** [comment]
@fricyo`zht.xml`

**Comment by fricyo** [comment]
谢谢 那我用对了

**Comment by Vdragon** [comment]
Seems to be a feature-wize counterpart of [Unlocker](http://www.emptyloop.com/unlocker)?

Might look at it, however as I'm not using MS Windows as my main working environment I can't give an ETA.

**Pull request merged**